### ISSUE TITLE
[FAST-23] The NVDA doesn't read the all items in the footer-menu when user try to mouse hover on them

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3104]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Try to mouse hover on the all items in the footer-menu

**Expected Result**
The NVDA should read the all items in the footer-menu when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read the all items in the footer-menu when user try to mouse hover on them

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6b62afda%2Dc894%2D4ce2%2Da924%2D74186e1c15dc&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fdoremihong%2FBug%5F9

**Comment by hokienick** [comment]
This ticket has been moved to #3120
